packages/shared-kernel/src/logger/ILogger.ts

```diff
@@ -2,8 +2,18 @@
  * Logger interface
  */
 export interface ILogger {
+  level: LoggerLevel;
```

Comment:
@cesarenaldi what do you think about extending ILogger like this and using this to determine debug mode deep in the code? see https://github.com/lens-protocol/lens-sdk/pull/780/files#diff-568da022b5078bcf9590f1377af950be03777482416db3e0d009de7b1fd32f25R52